odl/phantom/misc_phantoms.py

```diff
@@ -146,6 +146,82 @@ def rect(x):
     return out.ufuncs.minimum(1, out=out)
 
 
+def text(space, text, font='arial', scale=0.8, inverted=True):
+    """Create phantom of text.
+
+    The text is represented by an scalar image taking values in [0, 1].
+    Depending on the choice of font, the text may or may not be anti-aliased.
+    anti-aliased text can take any value between 0 and 1, while
+    non-anti-aliased text takes values exclusively in {0, 1}.
+
+    This method requires the PIL package.
```

Comment:
Recommend [`pillow`](https://python-pillow.org/) rather than the deprecated (AFAIK) PIL. (Import stays the same.)